ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/messages/snap/AccountRangeMessageTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
Update the copyright.
This comment applies also to other classes in this package.